kratos/mpi/tests/test_utilities/mpi_cpp_test_utilities.cpp

```diff
@@ -172,7 +174,101 @@ void MPICppTestUtilities::GenerateDistributedBarStructure(
         }
     }
 
-    // Compute communicaton plan and fill communicator meshes correctly
+    // Compute communication plan and fill communicator meshes correctly
+    auto filler = ParallelFillCommunicator(rModelPart, rDataCommunicator);
+    filler.Execute();
+}
+
+/***********************************************************************************/
+/***********************************************************************************/
+
+void MPICppTestUtilities::GenerateDistributedTriangleMesh(
+    ModelPart& rModelPart,
+    const DataCommunicator& rDataCommunicator
+    )
+{
+    // Set MPI coomunicator
+    ModelPartCommunicatorUtilities::SetMPICommunicator(rModelPart, rDataCommunicator);
```

Comment:
are you sure this is needed? I think the fillcomm does this

Reply:
It is possible

Reply:
Can you check? Does it work without?

Reply:
I will check